src/main/java/net/honux/chess/Color.java

```diff
@@ -0,0 +1,6 @@
+package net.honux.chess;
+
+public enum Color {
+    white,
+    black
```

Comment:
https://stackoverflow.com/questions/3069743/coding-conventions-naming-enums
`enum` 을 작성할 때는 대문자로 구성해주시는 것이 컨벤션이에요.

```suggestion
    WHITE,
    BLACK
```

Reply:
넵 수정하겠습니다.